darp_node:
  ros__parameters:
    example_parameter: "Hello, world!"